python/paddle/fluid/contrib/slim/unitest/test_quantization_pass.py

```diff
@@ -0,0 +1,204 @@
+#   copyright (c) 2018 paddlepaddle authors. all rights reserved.
+#
+# licensed under the apache license, version 2.0 (the "license");
+# you may not use this file except in compliance with the license.
+# you may obtain a copy of the license at
+#
+#     http://www.apache.org/licenses/license-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the license is distributed on an "as is" basis,
+# without warranties or conditions of any kind, either express or implied.
+# see the license for the specific language governing permissions and
+# limitations under the license.
+
```

Comment:
`test_quantization_pass.py` has been moved in the `python/paddle/fluid/contrib/slim/tests` dir. If this is not necessary， please remove it.